universal/build.gradle

```diff
@@ -33,7 +23,7 @@ class NettyEpollTransformer implements Transformer {
 
     boolean hasTransformedResource() { this.extractedNettyNative != null }
 
-    void modifyOutputStream(ZipOutputStream jos, boolean preserveFileTimestamps) {
+    void modifyOutputStream(shadow.org.apache.tools.zip.ZipOutputStream jos, boolean preserveFileTimestamps) {
         ZipEntry entry = new ZipEntry("META-INF/native/libcom_vexsoftware_votifier_netty_transport_native_epoll_x86_64.so")
         entry.time = TransformerContext.getEntryTimestamp(preserveFileTimestamps, entry.time)
         jos.putNextEntry(entry)
```

Comment:
Any reason why we then aren't using

```
plugins {
    id 'com.github.johnrengelman.shadow'
}
```

as documented in https://docs.gradle.org/current/userguide/plugins.html#sec:subprojects_plugins_dsl here?

Reply:
It is using the shadow plugin. It is likely a relocated dependency from shadow.